src/core/map/generator.rs

```diff
@@ -24,7 +24,7 @@ impl MapGenerator for SimpleAlternationMapGenerator {
 	}
 }
 impl SimpleAlternationMapGenerator {
-	pub fn new<'a, NameIter: IntoIterator, IO: EngineIO>(
+	pub fn new<NameIter: IntoIterator, IO: EngineIO>(
```

Comment:
Old leftovers, need to remove it, though this SimpleAlternationMapGenerator won't exist in time anyway.  ^.^